flytekit/annotated/node.py

```diff
@@ -36,7 +38,8 @@ def get_registerable_entity(self) -> SdkNode:
             raise Exception("Node flyte entity none")
 
         if self._flyte_entity._registerable_entity is None:
-            raise Exception("Node registerable entity has not been set")
+            print("WARNING!!!!!!!!!!!!!!!!!!!  not really")
```

Comment:
?

Reply:
i'm removing this if block entirely so this becomes fully recursive.  I was kinda worried about an infinite loop situation for some reason but I don't think that can happen... not unless new entities get created somehow, but at least then we'll run out of memory too.